### PR TITLE
refine doc `ata_elastic_scheduling.md `

### PR DESCRIPTION
**Pull request by skydoorkai**
Horovod also supports TensorFlow 2.x eager mode. Update the description.

### CODE REVIEW

File: docs/blogs/ata_elastic_scheduling.md [hunk 1]
```diff
@@ -56,69 +56,14 @@ ElasticDL 利用 TensorFlow eager execution 和 Kubernetes API，
...
-    loss = self._loss(outputs, labels)
-    # Add regularization loss if any
-    if self._model.losses:
-        loss += tf.math.add_n(self._model.losses)
-grads = tape.gradient(loss, self.get_trainable_items())
-```
-
-ElasticDL通过 tape 获取 gradient 后，
-可以通过 Parameter Server 或者 AllReduce 分布式策略来更新模型参数。
+然后收集 gradients，并且通过 AllReduce 调用汇聚 gradients 并且更新模型。在 TensorFlow 1.x graph mode 下， 深度学习计算是表示成一个计算图（graph）， 并且由 TensorFlow runtime 解释执行。 Horovod 通过包裹 Optimizer 的方式添加对 gradient 的 AllReduce 调用。 TensorFlow 2.x eager mode 采用和解释执行图完全不同的深度学习计算方式。 前向计算过程把对基本计算单元（operator）的调用记录在一个内存数据结构 tape 里， 随后反向计算过程（计算 gradients）可以回溯这个 tape， 以此调用 operator 对应的 gradient operator。 Horovod 通过包裹 tape 完成 AllReduce 调用。 Horovod 和 TensorFlow 一样，不是 Kubernetes-native，所以它提供的 AllReduce 操作不支持容错和弹性调度。这一点和 ElasticDL 不一样。ElasticDL 通过 tape 获取 gradient 后，可以使用 Parameter Server 或者 AllReduce 分布式策略来更新模型参数, 并且支持容错和弹性调度。
```
Comment (workingloong) [inline docs/blogs/ata_elastic_scheduling.md]: 最后一句：
并且，ElasticDL 作为 kubernetes-native 的分布式系统，可以通过调用 Kubernetes API 来支持容错和弹性调度。

File: docs/blogs/ata_elastic_scheduling.md [hunk 2]
```diff
@@ -130,13 +75,13 @@ Kubernetes API 来起止进程。ElasticDL 没有选择开发 Kubernetes Operato
...
 
-ElasticDL 的 master 会根据数据索引将数据分片，然后为每个数据分片创建一个 task。
+ElasticDL 的 master 会根据数据索引将数据分片，为每个数据分片创建一个 task。
 然后 master 会调用 Kubernetes API 启动多个 worker 进程。每个 worker 启动后，
 会向 master 请求 task。worker 收到来自 master 分发的 task 后，
 会读取 task 对应的数据分片来前向计算和梯度计算。
 
-同时，master 会通过 Kubernetes API 观察集群中每个worker的状态。
-当有 worker 被高优先级作业抢占后，master 会回收该 worker 的未完成task，
+同时，master 会通过 Kubernetes API 观察集群中每个 worker 的状态。
```
Comment (workingloong) [inline docs/blogs/ata_elastic_scheduling.md]: Kubernetes API 监听集群中每个 worker 的状态。

File: docs/blogs/ata_elastic_scheduling.md (hunk 1 above)
Comment (skydoorkai) [reply docs/blogs/ata_elastic_scheduling.md]: Changed.

File: docs/blogs/ata_elastic_scheduling.md (hunk 2 above)
Comment (skydoorkai) [reply docs/blogs/ata_elastic_scheduling.md]: Modified.

**Review by workingloong — APPROVED** [review]
Excellent!
